tests/test_sklearn_dict_vectorizer_converter.py

```diff
@@ -21,24 +21,19 @@ def test_model_dict_vectorizer(self):
         data = [{"amy": 1.0, "chin": 200.0}, {"nice": 3.0, "amy": 1.0}]
         model.fit_transform(data)
         model_onnx = convert_sklearn(
-            model,
-            "dictionary vectorizer",
+            model, "dictionary vectorizer",
             [(
                 "input",
                 DictionaryType(StringTensorType([1]), FloatTensorType([1])),
-            )],
-        )
+            )])
         self.assertTrue(model_onnx is not None)
         dump_data_and_model(
-            data,
-            model,
-            model_onnx,
+            data, model, model_onnx,
             basename="SklearnDictVectorizer-OneOff-SkipDim1",
             allow_failure="StrictVersion(onnxruntime.__version__)"
                           " <= StrictVersion('0.1.3') or "
                           "StrictVersion(onnx.__version__)"
-                          " < StrictVersion('1.3.0')",
-        )
+                          " < StrictVersion('1.3.0')")
```

Comment:
Why these changes, I think the previous version is better for adding new arguments to dump_data_and_model(), which would avoid modifying existing lines.

Reply:
Sorry, my screen is small :)